src/domains/queries.ml

```diff
@@ -252,6 +255,7 @@ struct
     | Any (WarnGlobal _) -> 35
     | Any (Invariant _) -> 36
     | Any (IterSysVars _) -> 37
+    | Any (InvariantGlobal _) -> 38
```

Comment:
All the queries (including this one) are in all the other places, because they've been ordered together into semantically related groups. They aren't in `order` because inserting inserting a new one semantically in the middle would require manual renumbering or placing the new largest number not at the end.